File: week2/exit.c
#include <cs50.h>
#include <stdio.h>

int main(int argc, string argv[])
{
    if (argc != 2)
    {
        printf("Missing command-line argument\n");
        // Return value of any no as uncussesful besides 0. 1 is commonly used
        return 1;
    }
    else
    {
        printf("hello, %s\n", argv[1]);
        // Return 0 meaning that all went well(sucessful)
        return 0;
    }
}

File: week6/part3/exit.py
from sys import argv, exit

if len(argv) != 2:
    print("Missing command-line argument")
    exit(1)               # Like return 1 for unsuccessful in C

print(f"Hello, {argv[1]}")
exit(0)                   # Like return 0 for success in C



# When importing te entire sys and not the specifics
"""
import sys

if len(sys.argv) != 2:
    print("Missing Command-Line Argument")
    sys.exit(1)

print(f"Hello, {sys.argv[1]}")
sys.exit(0)
"""